src/ray/protobuf/gcs.proto

```diff
@@ -42,10 +42,11 @@ enum TablePrefix {
   DIRECT_ACTOR = 17;
   // WORKER is already used in WorkerType, so use WORKERS here.
   WORKERS = 18;
-  INTERNAL_CONFIG = 29;
-  TABLE_PREFIX_MAX = 20;
-  PLACEMENT_GROUP_SCHEDULE = 21;
-  PLACEMENT_GROUP = 22;
+  INTERNAL_CONFIG = 19;
```

Comment:
Btw, internal config was 29 before lol,

Reply:
Changing its value should have no effect? Otherwise, TABLE_PREFIX_MAX will be smaller than INTERNAL_CONFIG.

Reply:
I meant it is a good catch. I am surprised how it worked before.